File: C#/1001 - Extremamente Básico.cs
/* 
   ==UserScript==
 @name		    Extremamente Básico
 @namespace   	https://github.com/Ddavidi/uri-online-judge-beecrowd
 @description 	URI 1001 - Extremamente Básico
 @copyright   	@ddavidi_
 @author        @ddavidi_	
   ==/UserScript==
*/

using System;

class URI {
    
  static void Main(string[] args) {
      
    int valueA = Convert.ToInt32(Console.ReadLine());
    int valueB = Convert.ToInt32(Console.ReadLine());
    int valueX = valueA + valueB;
    
    Console.WriteLine("X = " + valueX);
    
  }
}

File: Java/1001 - Extremamente Básico.java
/* 
   ==UserScript==
 @name         Extremamente Básico
 @namespace    https://github.com/Ddavidi/uri-online-judge-beecrowd
 @description  URI 1001 - Extremamente Básico
 @copyright    @ddavidi_
 @author       @ddavidi_	
   ==/UserScript==
*/

import java.io.IOException;
import java.util.Scanner; 

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner scan = new Scanner(System.in);
		int a = scan.nextInt();
		int b = scan.nextInt();
		
		int x = a + b;
		
		System.out.println("X = " + x);
 
    }
 
}
